src/AddressUkPafLookupViewBridge.js

```diff
@@ -0,0 +1,145 @@
+var bridge = function (presenterPath) {
+    window.rhubarb.viewBridgeClasses.HtmlViewBridge.apply(this, arguments);
+};
+
+bridge.prototype = new window.rhubarb.viewBridgeClasses.HtmlViewBridge();
+bridge.prototype.constructor = bridge;
+
+bridge.prototype.attachEvents = function() {
+
+    var self = this,
+        alertClass = "c-alert",
+        manualAddressElements = $(".manual-fields"),
+        searchAddressElement = $(".search-fields"),
+        insertManualAddressLink = $(".manual-address-link"),
+        manualAddressPar = $(".manual-address-par"),
+        searchLink = $(".search-address-link"),
+        houseNumber = self.findChildViewBridge('HouseNumber'),
+        postCodeSearch = self.findChildViewBridge('PostCodeSearch'),
+        searchError = $(".search-error"),
+        searchResultsMsg = $(".search-results-msg"),
+        resultItemsList = $(".search-results-items"),
+        searchButton = self.findChildViewBridge('Search'),
+        // address fields
+        line1 = self.findChildViewBridge('Line1'),
+        line2 = self.findChildViewBridge('Line2'),
+        town = self.findChildViewBridge('Town'),
+        county = self.findChildViewBridge('County'),
+        postCode = self.findChildViewBridge('PostCode'),
+        addressProperties = ['AddressLine1', 'AddressLine2', 'Town', 'County', 'Postcode'];
+
+    // hide spinner on loading
+    if (typeof spinnerGif != 'undefined') {
+        spinnerGif.hide();
```

Comment:
This variable isn't defined anywhere in this module. I'm assuming it's something which you've made globally available from the JS of the donation form for fields of life, so I've left it present to avoid breaking that, but I've put a check for the variable actually existing to prevent a JS error on other projects that use this. In any case, this kind of coupling directly to variables declared in another repository is very bad code.

I'd like to change the way the spinner animation works entirely because Paul can use Rhubarb's built-in CSS to display it without us having JS to hide and show it anyway.